server/environment.yaml

```diff
@@ -133,11 +153,12 @@ dependencies:
       - tomli==2.0.1
       - tomlkit==0.12.1
       - tornado==6.3.3
+      - tqdm==4.66.1
       - traitlets==5.9.0
       - typing-extensions==4.7.1
       - tzdata==2023.3
       - uri-template==1.3.0
-      - urllib3==2.0.4
+      - urllib3
```

Comment:
Any reason there's not a specific version for this package after the change?

Reply:
Google cloud was rejecting urllib3==2.0.4, so I just changed it to the default version